File: mysql/java/create/mysql_create.java
// ---------------------------------------------------------------------
/*
	java/create/mysql_create.java

					Sep/13/2013

*/
// ---------------------------------------------------------------------
import	java.util.HashMap;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.Statement;
import java.sql.SQLException;

// ---------------------------------------------------------------------
public class mysql_create
{
// ---------------------------------------------------------------------
public static void main (String args [])
	throws SQLException,
	InstantiationException, IllegalAccessException
{
	System.out.println ("*** 開始 ***");

	HashMap <String, HashMap <String,String>> dict_aa
			= data_prepare_proc ();
	try
	{
	String url="jdbc:mysql://host_mysql/city";
	Connection conn = DriverManager.getConnection (url,"scott","tiger");

	Statement ss = conn.createStatement ();

	rdb_common.drop_table_proc (ss);
	rdb_common.create_table_proc (ss);

	rdb_common.display_proc (conn);

	rdb_common.dict_to_db_proc (dict_aa,ss);

	ss.close ();

	conn.close();
	}
	catch (Exception ee)
		{
		ee.printStackTrace();
		}


	System.out.println ("*** 終了 ***");
}

// ---------------------------------------------------------------------
static HashMap <String, HashMap <String,String>>
data_prepare_proc ()
{
	HashMap <String, HashMap<String,String>> dict_aa
		= new HashMap <String, HashMap<String,String>> ();

	dict_aa = text_manipulate.dict_append_proc
		(dict_aa,"t3321","岡山",91476,"2008-10-12");

	dict_aa = text_manipulate.dict_append_proc
		(dict_aa,"t3322","倉敷",72849,"2008-4-23");

	dict_aa = text_manipulate.dict_append_proc
		(dict_aa,"t3323","津山",98253,"2008-5-24");

	dict_aa = text_manipulate.dict_append_proc
		(dict_aa,"t3324","玉野",87624,"2008-9-14");

	dict_aa = text_manipulate.dict_append_proc
		(dict_aa,"t3325","笠岡",72531,"2008-8-12");

	dict_aa = text_manipulate.dict_append_proc
		(dict_aa,"t3326","井原",28539,"2008-7-28");

	dict_aa = text_manipulate.dict_append_proc
		(dict_aa,"t3327","総社",39457,"2008-6-19");

	dict_aa = text_manipulate.dict_append_proc
		(dict_aa,"t3328","高梁",47365,"2008-11-15");

	dict_aa = text_manipulate.dict_append_proc
		(dict_aa,"t3329","新見",56874,"2008-10-24");

	return	dict_aa;
}
// ---------------------------------------------------------------------
}

// ---------------------------------------------------------------------

File: mysql/scala/create/mysql_create.scala
// -----------------------------------------------------------------
//	create/mysql_create.scala
//
//						Jul/31/2011
// -----------------------------------------------------------------
import scala.io.Source
import java.sql._

// -----------------------------------------------------------------
object mysql_create
{
// -----------------------------------------------------------------
def main (args: scala.Array[String])
{
	println	("*** 開始 ***")

	val str_connect= "jdbc:mysql://host_mysql/city"

	val conn = DriverManager.getConnection (str_connect,"scott","tiger")

	val sss = conn.createStatement ()

	sql_manipulate.drop_table_proc (sss)
	sql_manipulate.create_table_proc (sss)

	insert_data_proc (sss)

	sql_manipulate.display_proc (conn)

	sss.close
	conn.close

	println	("*** 終了 ***")
}


// -----------------------------------------------------------------
def insert_data_proc (sss:Statement)
{
	sql_manipulate.insert_record_proc (sss,"t3321","岡山",41357,"1998-7-11")
	sql_manipulate.insert_record_proc (sss,"t3322","倉敷",34891,"1998-6-24")
	sql_manipulate.insert_record_proc (sss,"t3323","津山",56592,"1998-5-9")
	sql_manipulate.insert_record_proc (sss,"t3324","玉野",52481,"1998-9-29")
	sql_manipulate.insert_record_proc (sss,"t3325","笠岡",19237,"1998-10-16")
	sql_manipulate.insert_record_proc (sss,"t3326","井原",49784,"1998-11-24")
	sql_manipulate.insert_record_proc (sss,"t3327","総社",58296,"1998-12-30")
	sql_manipulate.insert_record_proc (sss,"t3328","高梁",41284,"1998-11-18")
	sql_manipulate.insert_record_proc (sss,"t3329","新見",58347,"1998-7-21")
}

// -----------------------------------------------------------------
}
// -----------------------------------------------------------------
